src/main/java/org/broadinstitute/consent/http/service/DataAccessRequestService.java

```diff
@@ -245,6 +245,8 @@ private Document createDocumentFromDar(DataAccessRequest d) {
         Document document = Document.parse(gson.toJson(d.getData()));
         document.put(DarConstants.DATA_ACCESS_REQUEST_ID, d.getId());
         document.put(DarConstants.REFERENCE_ID, d.getReferenceId());
+        document.put(DarConstants.CREATE_DATE, d.getCreateDate());
+        document.put(DarConstants.SORT_DATE, d.getCreateDate());
```

Comment:
This line should be: 
```suggestion
        document.put(DarConstants.SORT_DATE, d.getSortDate());
```

Reply:
I think we also need to make a couple similar updates to `public DataAccessRequest insertDraftDataAccessRequest(User user, DataAccessRequest dar)`. Can you add these lines just before the dao insert call?
```
        SimpleDateFormat sdf = new SimpleDateFormat("yyyy-MM-dd");
        dar.getData().setCreateDate(now.getTime());
        dar.getData().setPartialDarCode("temp_DAR_" + sdf.format(now));
```

Reply:
While we're at it, we could abstract `"temp_DAR_"` into a public static string so both classes can refer to it.